options/rtdl/generic/main.cpp

```diff
@@ -108,10 +108,12 @@ extern "C" {
 	}
 }
 
-extern "C" [[gnu::alias("dl_debug_state"), gnu::visibility("default")]] void _dl_debug_state();
+#ifndef __clang__
+	extern "C" [[gnu::alias("dl_debug_state"), gnu::visibility("default")]] void _dl_debug_state();
+#endif
 
 // This symbol can be used by GDB to find the global interface structure
-[[gnu::visibility("default")]] DebugInterface *_dl_debug_addr = &globalDebugInterface;
+[[ gnu::visibility("default") ]] DebugInterface *_dl_debug_addr = &globalDebugInterface;
```

Comment:
I think the spaces here collide with current convention, but I'm not sure

Reply:
Hmm, I noticed a mix of spaces and not spaces within existing code, that was changed because I had to add some gnu::visibility lines back

Reply:
I think for the most part we use no spaces